doc/api/tls.markdown

```diff
@@ -77,6 +77,51 @@ handshake extensions allowing you:
   * SNI - to use one TLS server for multiple hostnames with different SSL
     certificates.
 
+## Modifying the Default TLS Cipher suite
+
+Node.js is built with a default suite of enabled and disabled TLS ciphers.
+Currently, the default cipher suite is:
+
+    ECDHE-RSA-AES128-GCM-SHA256:
```

Comment:
Well, yes, it's a delimiter that's part of the cipher list format supported by openssl. There may be a better way of representing that list, but I'd rather make that change independent of this one.